packages/circuit-ui/components/PhoneNumberInput/PhoneNumberInput.mdx

```diff
@@ -7,11 +7,79 @@ import * as Stories from './PhoneNumberInput.stories';
 
 <Status variant="experimental" />
 
-TODO: Write short summary
+The PhoneNumberInput component allows users to input phone numbers.
+It provides a user-friendly interface for entering phone numbers, including country code selection and subscriber number.
+
 
 <Story of={Stories.Base} />
 <Props />
 
-## Usage
+## When to use it
+
+When you need to collect a phone number from the user. This could be for contact information, two-factor authentication, or other purposes where a phone number is required.
+The component allows the user to select their country code, and normalizes the input to the E.164 format.
+
+## Usage guidelines
+
+- **Do** always accompany the `PhoneNumberInput` field with a visible label. This helps users understand what information is required.
+- **Do not** use `PhoneNumberInput` for non-phone number data. It is specifically designed for phone number input and includes specific formatting and validation for phone numbers.
+- **Do not** communicate that a `PhoneNumberInput` field is required on the label or the placeholder. We assume that everything is required for the user unless explicitly said so.
```

Comment:
```suggestion
- **Do not** communicate that a `PhoneNumberInput` field is required on the label or the placeholder. We assume that everything is required for the user unless explicitly stated otherwise.
```